source/process/sg_rest_markup.rst

```diff
@@ -86,23 +96,13 @@ Create numbered lists and procedure steps using arabic numerals for the top-leve
 Links to external web pages
 ---------------------------
 
-The quickest and easiest way to create a link in reStructuredText is to use the URL without any extra markup. For example:
-
-.. code-block:: none
-  
-  https://www.mattermost.org/manifesto/
-
-On output to HTML, the link looks like this: https://www.mattermost.org/manifesto/.
-
-You can also create a link that has link text. For example:
```

Comment:
Yes. "Naked" links look messy in my mind, I think we should try to embed them in the text unless there is a reason to show the link.